opm/material/common/UniformXTabulated2DFunction.hpp

```diff
@@ -379,7 +393,127 @@ class UniformXTabulated2DFunction
             os << "\n";
         }
     }
+    void extendTableStart(){
```

Comment:
some comment what this method does and how it does is required before merge. Also: can it be made private?